packer/scylla_install_image

```diff
@@ -46,6 +46,38 @@ def half_of_diskfree():
     disk = os.statvfs('/')
     return int((disk.f_bavail * disk.f_frsize) / 2)
 
+
+def install_required_packages(distro):
+    """
+    The python packages will be installed with easy_install instead of pip, because
+    easy_install requires fewer dependencies, no need to install gcc, it makes our AMI smaller.
+    """
+
+    deb_packages = ['python3-pystache', 'python3-lockfile', 'python3-setuptools']
+    rpm_packages = ['pystache', 'python-daemon', 'python-setuptools']
+    pip_packages = ['scylla-api-client']
+
+    if distro == 'centos':
+        run('yum install -y epel-release')
+
+        for package in rpm_packages:
+            run(f'yum install -y {package}')
+
+        for package in pip_packages:
+            run(f'python -m easy_install {package}')
+
```

Comment:
Why are we installing some packages via rpm and some packages via pip? It's a recipe for confusion and non-upgraded packages.

Reply:
As descussed  above
- aws-cfn-bootstrap - no deb package so we use the provided tar with easy_install / pip
- scylla-api-client - a python project built as a python package. we might later add/switch to a deb package